### PR TITLE
Feature/broker

### PR DESCRIPTION
**Pull request by jbygdell**
Drop in replacement for the broker code in `sda-pipeline`.
Note the absence of the `VerifyJSON` function as that should be handled differently and in a different place.

### CODE REVIEW

File: .github/workflows/broker.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+on: [push]
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: [1.18, 1.19]
```
Comment (blankdots) [inline .github/workflows/broker.yaml]: this might come into conflict with #3

**Review by blankdots — APPROVED** [review]
this seems to reflect what is in the sda-pipeline, with some minor changes

File: .github/workflows/broker.yaml (hunk 1 above)
Comment (jbygdell) [reply .github/workflows/broker.yaml]: GO 1.16 & 1.17 are [EOL](https://endoflife.date/go)

**Comment by jbygdell** [comment]
Rebased on Main

**Review by blankdots — APPROVED** [review]

**Review by dbampalikis — APPROVED** [review]

**Review by dbampalikis — APPROVED** [review]

**Review by norling — APPROVED** [review]

**Pull request merged**